src/ray/raylet/node_manager.cc

```diff
@@ -650,6 +683,12 @@ void NodeManager::HeartbeatAdded(const ClientID &client_id,
                   << client_id;
     return;
   }
+
+  // Trigger local GC at the next heartbeat interval.
+  if (heartbeat_data.should_global_gc()) {
+    should_local_gc_ = true;
```

Comment:
Could we just `DoLocalGC` here and not have the `should_local_gc_` flag?

Reply:
Oh never mind, I guess we might receive more than one `HeartbeatAdded` per interval?

Reply:
Yeah. Actually I need to change the other site to set the flag too.